apps/emqx/src/emqx_schema.erl

```diff
@@ -216,15 +215,14 @@ roots(high) ->
                     importance => ?IMPORTANCE_HIDDEN
                 }
             )},
-        {?EMQX_AUTHENTICATION_CONFIG_ROOT_NAME, authentication(global)},
         %% NOTE: authorization schema here is only to keep emqx app pure
         %% the full schema for EMQX node is injected in emqx_conf_schema.
         {?EMQX_AUTHORIZATION_CONFIG_ROOT_NAME,
             sc(
                 ref(?EMQX_AUTHORIZATION_CONFIG_ROOT_NAME),
                 #{importance => ?IMPORTANCE_HIDDEN}
             )}
-    ];
+    ] ++ emqx_schema_hooks:injection_point('roots.high');
```

Comment:
please keep this at the old position in the list.

Reply:
✅